Tests/Testo/Invoice/PaymentInformation/BitPayPaymentServiceTest.php

```
@@ -0,0 +1,256 @@
+        return new BitPayPaymentService(
+            $this->makeSettingRepository($sandbox),
+            $this->makeLoggerInterfaceSpy(),
+            $this->makeHttpClient($mock),
+        );
+    }
+
+    public function getDriverKeyReturnsBitpay(): void
+    {
+        $service = $this->makeService(new MockHandler([]));
+
+        Assert::same('bitpay', $service->getDriverKey());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Testo\Assert' in method 'getDriverKeyReturnsBitpay'. (undefined)

(StaticAccess)